src/main/java/com/khomishchak/cryproportfolio/configurations/JwtFilterConfig.java

```diff
@@ -0,0 +1,71 @@
+package com.khomishchak.cryproportfolio.configurations;
+
+import com.khomishchak.cryproportfolio.services.security.JwtService;
+
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.web.authentication.WebAuthenticationDetailsSource;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+import io.netty.util.internal.StringUtil;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+public class JwtFilterConfig extends OncePerRequestFilter {
+
+    private static final String AUTHORIZATION_HEADER = "Authorization";
+
+    private final UserDetailsService userDetailsService;
+    private final JwtService jwtService;
+
+    public JwtFilterConfig(UserDetailsService userDetailsService, JwtService jwtService) {
+        this.userDetailsService = userDetailsService;
+        this.jwtService = jwtService;
+    }
+
+    @Override
+    protected void doFilterInternal(HttpServletRequest request,
+            HttpServletResponse response,
+            FilterChain filterChain) throws ServletException, IOException {
+
+        String jwtTokenHeader = request.getHeader(AUTHORIZATION_HEADER);
```

Comment:
FYI: `There is a static constant org.springframework.http.HttpHeaders.AUTHORIZATION`